--- theme.inc.orig	2013-03-29 10:14:24.000000000 -0700
+++ theme.inc	2013-07-31 22:20:44.000000000 -0700
@@ -698,7 +698,7 @@ function template_preprocess_views_view_
     }
     if ($row) {
       // Fill up the last line only if it's configured, but this is default.
-      if (!empty($handler->options['fill_single_line']) && count($rows)) {
+      if (!empty($handler->options['fill_single_line']) && $row_count==count($rows)) {
         for ($i = 0; $i < ($columns - $col_count); $i++) {
           $row[] = '';
         }
