diff --git a/date_ical_plugin_row_ical_feed.inc b/date_ical_plugin_row_ical_feed.inc
index 7181811..84c5476 100644
--- a/date_ical_plugin_row_ical_feed.inc
+++ b/date_ical_plugin_row_ical_feed.inc
@@ -152,7 +152,7 @@ class date_ical_plugin_row_ical_feed extends views_plugin_row {
       return;
     }
 
-    // Set the item date to the proper display timezone;
+    // Set the item timezone for doing the all day test.
     $start->setTimezone(new dateTimezone($item['timezone']));
     $end->setTimezone(new dateTimezone($item['timezone']));
 
@@ -161,6 +161,10 @@ class date_ical_plugin_row_ical_feed extends views_plugin_row {
 
     $all_day = date_is_all_day($start, $end, date_granularity_precision($info['granularity']));
 
+    // Guarantee an always accurate time by using UTC.
+    $start->setTimezone(new dateTimezone('UTC'));
+    $end->setTimezone(new dateTimezone('UTC'));
+
     // According to RFC 2445 (clarified in RFC 5545) the DTEND value is
     // non-inclusive.  When it is a DATE rather than a DATETIME, this means
     // that we should add one day to its value.
